docs/flank-output-investigation/flank_output_propositon.md

```diff
@@ -0,0 +1,101 @@
+  Uploading [android_shards.json] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/2021-03-01_13-01-56.722225_lYrQ/...
+```
+
+### Uploading files
+
+```
+[Uploading files]
+
+  Uploading [app-debug.apk] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/2021-03-01_13-01-56.722225_lYrQ/...
+  Uploading [app-single-success-debug-androidTest.apk] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/2021-03-01_13-01-56.722225_lYrQ/...
+  Uploading [session_id.txt] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/2021-03-01_13-01-56.722225_lYrQ/...
+```
+
+### Matrix info
+
+```
+[Matrix info]
+	Found 1 test / 1 shard
+
+  1 matrix ids created in 0m 5s
+  
+   matrix-1kozhsv2imkru https://console.firebase.google.com/project/flank-open-source/testlab/histories/bh.da0c237aaa33732/matrices/8233077922466140188/executions/bs.d3f60304f671ce86
+```
+
+
+
+### Storage information
+
+```bash
+
+  Raw results will be stored in your GCS bucket at [https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/2021-03-01_13-01-56.722225_lYrQ]
+
+```
+
+### Test status
+
+```bash
+[Execution statuses]
+  3m 11s Test executions status: FINISHED:1
+  3m 11s matrix-1kozhsv2imkru FINISHED
+
+```
+
+### Cost report section
+
+```bash
+[CostReport]
+  Virtual devices
+    $0.02 for 1m
+
+  Uploading [CostReport.txt] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/2021-03-01_13-01-56.722225_lYrQ/...
+
+```
+
+### Results section
+
+```bash
+
+[MatrixResultsReport]
+  1 / 1 (100.00%)
+
+┌─────────┬──────────────────────┬────────────────────────────┬────────────────────────────────┐
+│ OUTCOME │      MATRIX ID       │      TEST AXIS VALUE       │          TEST DETAILS          │
+├─────────┼──────────────────────┼────────────────────────────┼────────────────────────────────┤
+│ success │ matrix-1kozhsv2imkru │ NexusLowRes-28-en-portrait │ 1 test cases passed, 1 skipped │
+└─────────┴──────────────────────┴────────────────────────────┴────────────────────────────────┘
+
+  Uploading [MatrixResultsReport.txt] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/2021-03-01_13-01-56.722225_lYrQ/...
+  Uploading [JUnitReport.xml] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/2021-03-01_13-01-56.722225_lYrQ/...
+  Uploading [matrix_ids.json] to https://console.developers.google.com/storage/browser/test-lab-v9cn46bb990nx-kz69ymd4nm9aq/2021-03-01_13-01-56.722225_lYrQ/...
+
+  matrix-1kozhsv2imkru https://console.firebase.google.com/project/flank-open-source/testlab/histories/bh.da0c237aaa33732/matrices/8233077922466140188/executions/bs.d3f60304f671ce86
```

Comment:
I do not like indentations in console outputs. IMO it looks sloppy especially when comes to fold long lines on narrow screens. Take a look at Gclouds output, they have almost no indentations.
WDYF?

Reply:
I think reduce indentations is a good idea also integration tests should be easier to update without indentations.